client/lib/plans/features-list.tsx

```diff
@@ -302,7 +302,6 @@ import {
 	FEATURE_GROUP_PAYMENT_TRANSACTION_FEES,
 	FEATURE_COMMISSION_FEE_STANDARD_FEATURES,
 	FEATURE_COMMISSION_FEE_WOO_FEATURES,
-	FEATURE_PERSONAL_THEMES,
```

Comment:
We can remove the `FEATURE_PERSONAL_THEMES` feature from `calypso-products` since it was created just for this specific use case (which we're now deleting).